deploy/minikube-ci.sh

```diff
@@ -23,7 +23,6 @@ echo "Tekton controller is running"
 
 #CRDS are sometimes racey
 kubectl apply -k $DIR/crds/base
-kubectl apply -f https://raw.githubusercontent.com/openshift/api/master/quota/v1/0000_03_quota-openshift_01_clusterresourcequota.crd.yaml
 sleep 2
```

Comment:
Does anyone know why we need these `sleep` commands after `kubectl apply`? I know it wasn't added in this PR, but I am curious.

Reply:
Minikube is slow to load the CRDs